tasks.py

```
@@ -1358,8 +1358,14 @@ def e2etest(
         echo=True,
     )
 
-    if export != None:
-        run("{} export logs {}".format(kind_path, export), warn=True)
+    # kind export logs only applies when e2etest is running against a kind
+    # cluster. OpenShift CI (and other external clusters) pass --kubeconfig
+    # explicitly and must not call into kind here.
+    if export != None and kubeconfig is None:
```

Ruff (0.16.3)

[error] Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)